client/actions/dispatchers.js

```diff
@@ -2,6 +2,7 @@ import logoutAction from "./logout";
 import {
   SET_AUTHENTICATION_STATUS,
   SET_MOBILE_PHONE_VERIFICATION_STATUS,
+  SET_USER_BANNED_STATUS,
```

Comment:
for consistency with the backend, wouldn't it be clearer if we called this `IS_ACTIVE` instead of `BANNED`?